BrandonBeltran/hackerrank/03/JavaCurrencyFormatterLegacy.java

```diff
@@ -0,0 +1,78 @@
+package bootcamp;
+
+import java.util.List;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.util.stream.Collectors;
+
+
+public class JavaCurrencyFormatterLegacy {
+
+    private static final String EMPTY_STRING = "";
+    private static final String US_CURRENCY_SIGN = "$";
+    private static final String INDIA_CURRENCY_SIGN = "Rs.";
+    private static final String CHINA_CURRENCY_SIGN = "￥";
+    private static final String FRANCE_CURRENCY_SIGN = "€";
+
+    public static void main(String[] args) {
+
+        /* This solution was made before I realized NumberFormat.getCurrencyInstance() existed.
+        * It's preserved here just for completion purposes */
+
+
+        Scanner scanner = new Scanner(System.in);
+        double payment = scanner.nextDouble();
+        scanner.close();
+
+        String [] paymentString = String.valueOf(payment).split("\\.", 0);
+        String decimals = paymentString[1].substring(0,2);
+        String numerals = paymentString[0];
+
+        List<String> splittedString = splitEveryN(numerals, 3);
+
+        String commaSeparated = splittedString.stream().map(Object::toString).
+                collect(Collectors.joining(",")).toString();
+        String spaceSeparated = commaSeparated.replaceAll(",", " ");
+
+        String us = US_CURRENCY_SIGN+commaSeparated+"."+decimals;
```

Comment:
Separate the + by a space
Why do you do it in some places and not others? Homologous your code, change it in all the code